examples/WORKSPACE.bazel

```diff
@@ -26,6 +26,42 @@ load("//deps:deps_jvm_external.bzl", "deps_jvm_external")
 
 deps_jvm_external()
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```

Comment:
I started to do the deps.bzl files like the other things do, but I think that would require I do 4, or pin the versions of each of these deps in the main deps file, which seemed unnecessary. happy to do whatever here though

Reply:
@UebelAndre do you have an opinion on this?

Reply:
I think this is ok but we should then come up with a new strategy for selecting what targets should run on which platforms. Maybe it's time to roll out `exec_compatible_with` or `target_compatible_with` instead of having `examples/third_party` be it's own package. But for the time being, can these dependencies be added to [examples/third_party/WORKSPACE.bazel](https://github.com/bazelbuild/rules_foreign_cc/blob/4ee863c1fc88b554b9b185f192eaadd2e44c011e/examples/third_party/WORKSPACE.bazel) as well?